substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/jdk/JMXInitializationFeature.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2022, 2022, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2022, 2022, Red Hat Inc. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package com.oracle.svm.hosted.jdk;
+
+import org.graalvm.nativeimage.hosted.Feature;
+import com.oracle.svm.core.annotate.AutomaticFeature;
+
+@AutomaticFeature
+public class JMXInitializationFeature implements Feature{
+    @Override
+    public void beforeAnalysis(BeforeAnalysisAccess access) {
+
+        try {
+            com.oracle.svm.core.jdk.NativeLibrarySupport.singleton().preregisterUninitializedBuiltinLibrary("management_agent");
```

Comment:
These registrations are successful but there is still a linking error at runtime when executing `FileSystemImpl.isAccessUserOnly0(String)`